bin/disco_ssh.py

```diff
@@ -13,6 +13,7 @@
      -h --help              Show this screen
      --debug                Log in debug level
      --env ENV              Environment to operate in
+     --first                In case of multiple matching instances, connect to the first instead of failing
```

Comment:
I don't really care one way or the other—picking `instances[0]` was easier than randomizing, and I didn't see any reason to pretend that I was randomizing internally.